test/config/default/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+# Adds namespace to all resources.
+namespace: test-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: test-
+
+# Labels to add to all resources and selectors.
+#commonLabels:
+#  someName: someValue
+
+# Each entry in this list must resolve to an existing
+# resource definition in YAML.  These are the resource
+# files that kustomize reads, modifies and emits as a
+# YAML string, with resources separated by document
+# markers ("---").
+resources:
+- ../rbac/*.yaml
+- ../manager/*.yaml
```

Comment:
crds/*.yaml

Reply:
Not adding CRDs because name-prefix transformer changes the CRD names.